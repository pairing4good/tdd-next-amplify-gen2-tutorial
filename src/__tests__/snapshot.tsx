
import RootLayout from "@/app/layout";
import Home from "@/app/page";
import { render } from "@testing-library/react";

it("renders homepage unchanged", () => {
  jest.spyOn(console, 'error').mockImplementation(jest.fn());
  const { container } = render(<RootLayout><Home /></RootLayout>);
  expect(container).toMatchSnapshot();
});
