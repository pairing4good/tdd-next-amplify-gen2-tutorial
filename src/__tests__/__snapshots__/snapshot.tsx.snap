// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <html
    lang="en"
  >
    <body
      class="className"
    >
      <h1>
        TDD Next Template
      </h1>
    </body>
  </html>
</div>
`;
